locale/Bengali.json

```diff
@@ -94,6 +94,13 @@
       "searchHint": "অনুসন্ধান করুন",
       "continueLabel": "নির্বাচন করুন এবং চালিয়ে যান"
     },
+    "hashMarkInformation": {
+      "CreatedBy": "দ্বারা সৃষ্টি",
+      "IssuedBy": "প্রদান করেছেন",
+      "CreatedAt": "তৈরি",
+      "HashLabel": "হ্যাশ",
+      "Transaction": "লেনদেন"
```

Comment:
@sankarshanmukhopadhyay @priyashankarbanerjee is this OK here?

Reply:
Need Check For Malayalam Also

Reply:
I think I can get this improved today.

Reply:
"CreatedAt": "সময়ে নির্মিত",

a single string change.